config-template.yaml

```
@@ -2,3 +2,4 @@ config:
   BinderHub:
     use_registry: true
     image_prefix: <docker-id>/<prefix>-
+    hub_url: http://<jupyterhub-ip>
```

```suggestion
    hub_url: http://<jupyterhub-ip>/
```
Does this need a trailing slash?

---

I don't think so, I've never used one